readme.md

```diff
@@ -77,82 +123,178 @@ View a snapshot of the current file's dependency tree (as a JSON file)
 1. Within a file, right click to open the context menu
 2. Click on `Dependents -> View this file's dependency tree`
 
-### Configuring settings
+### Bindings
 
-* `root`: the location where your JS files reside.
-* `config`: (Optional) the path to your requirejs configuration file
-* `sass_root`: (Optional) the path to your Sass files
-* `exclude`: (Optional) a list of paths and/or folder names to exclude from the search for dependents
- * Omitting folders that contain 3rd party libraries can drastically speed up the search for a large codebase.
- * The following folders are excluded by default: `node_modules`, `bower_components`, `vendor`
- * Note: Subdirectories are not supported; you can't supply `some/sub/folder` as an exclusion
-* `build_config`: (Optional) path to your RequireJS Build configuration json file
- * This should have a "modules" section that lists your bundles (entry points)
- * Supplying this yields a significant performance speedup when finding relevant app entry points
-* `node_path`: (Optional) path to your node installation. Defaults to `/usr/local/bin` on OSX
+To more swiftly and conveniently trigger the package's commands both key and mouse bindings are provided.
 
-You can define these settings on a per-project basis by creating
-a `.deprc` file in the main directory of your codebase.
+#### Key bindings
 
-Configure the settings above within the `.deprc` file by adding:
+By default, the following key bindings have been supplied:
 
-```js
-{
-  "root": "public/assets/js",
-  "config": "public/assets/js/config.js",  # Optional
-  "sass_root": "public/assets/sass",       # Optional
-  "exclude": ['jquery.js', 'require.js'],  # Optional
-  "build_config": "public/assets/js/build.json", # Optional
-  "node_path": "/my/node/install/folder"   # Optional
-}
-```
+OSX:
 
-Alternatively, you can specify the settings by going to
+* Jump to dependency: `Option + Command + Down arrow` 
+* Find Dependents: `Option + Command + Up arrow`
 
-`Preferences -> Package Settings -> Dependents -> Settings - User`
+  
+Windows and Linux:
+
+* Jump to dependency: `Ctrl + Shift + Down arrow`
+* Find Dependents: `Ctrl + Shift + Up arrow`
 
-and adding your configuration.
 
-#### Default key bindings
+
+#### Mouse bindings
 
 By default, the following key bindings have been supplied:
 
 OSX:
 
-* Find Dependents: `CMD + Option + Up arrow`
-* Jump to dependency: `CMD + Option + Right arrow` or `CMD + Option + Click`
+* Jump to dependency: `Option + Command + Click` on the dependency item
```

Comment:
I think the `Command` key being listed first is the norm for many OSX apps and ST plugins.